examples/kubernetes/connectivity-check/connectivity-check-proxy.yaml

```diff
@@ -879,6 +879,8 @@ spec:
 apiVersion: v1
 kind: Service
 
+---
+
```

Comment:
Seems like there's some extraneous YAML here. Is it possible to avoid these?

Reply:
Yeah, I think the `echo-c` gets defined twice here too. Probably just need to rework the way the echo-c is generated / integrated into these broader YAMLs.